### PR TITLE
Swap the order of two tips to make more sense

### PR DESCRIPTION
**Pull request by dholbert**
This just reorders things so that we show the tip that explains ETP (enhanced tracking protection) before the tip that explains the ETP Allowlist, instead of after.

Fixes https://github.com/mozilla-mobile/focus-android/issues/6912

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

**Review by jonalmeida — APPROVED** [review]

**Comment by dholbert** [comment]
Thanks for the review @jonalmeida !

What's the next step here to get this landed?  Is there anything I should do at this point?

**Comment by jonalmeida** [comment]
From the internal thread, it looks like we're waiting on product confirmation, but I'm tempted to land this since it makes logical sense.

**Comment by jonalmeida** [comment]
Another thing that needs to be updated before we land this are the UI tests `turnOffAndOnHomeScreenTipsTest`, `homeScreenTipsTest`, and `verifyAllowListLinkFromTips`.

**Comment by amedyne** [comment]
@jeffreygee Can you check from a UX perspective?

@jonalmeida Other than last look from UX, I agree that we can go ahead and land this.

**Comment by jeffreygee** [comment]
@jonalmeida makes sense to me, too! thanks

**Comment by jonalmeida** [comment]
CI PR: https://github.com/mozilla-mobile/focus-android/pull/6921

**Pull request merged**
